packages/network/lib/http-utils.ts

```diff
@@ -9,3 +9,18 @@ const NO_BODY_STATUS_CODES = [204, 304]
 export function responseMustHaveEmptyBody (req: IncomingMessage, res: IncomingMessage) {
   return _.includes(NO_BODY_STATUS_CODES, res.statusCode) || (req.method && req.method.toLowerCase() === 'head')
 }
+
+/**
+ * HTTP options to make Node.js's HTTP libraries behave as leniently as possible.
+ *
+ * These should be used whenever Cypress is processing "real-world" HTTP requests - like when setting up a proxy
+ * server or sending outgoing requests.
+ */
+export const lenientOptions = {
+  // increase header buffer for incoming response (ClientRequest) request (Server) headers, from 16KB to 1MB
+  // @see https://github.com/cypress-io/cypress/issues/76
+  maxHeaderSize: 1024 ** 2,
+  // allow requests which contain invalid/malformed headers
+  // https://github.com/cypress-io/cypress/issues/5602
+  insecureHTTPParser: true,
```

Comment:
i've been waiting for this change to come to Electron ever since i saw talk of it on Node's repo like a year ago, i hated having to launch Cylectron with NODE_OPTIONS

Reply:
Very nice. Looks like this enabled cleaning things up a lot. Love it when we can delete code 💯